addons/html_builder/static/tests/block_tab.test.js

```diff
@@ -0,0 +1,69 @@
+import { expect, test } from "@odoo/hoot";
```

Comment:
I would either add this test in 
- `/website/static/tests/builder/block_tab/` (either in one existing file which already tests the drag and drop or in a new file but with a better/more specific name), or in
-  `website/static/tests/builder/operation.test.js` but with a more generic flow, like call next on something async with shouldIntercept at true, then click, then check that the containers are active

I prefer the second choice: as long as it works as expected, we know that the drag and drop flow will work too, no need to test it specifically :slightly_smiling_face: 

(you won't need the stylecontent support in that case :wink:)